source/index.rst

```diff
@@ -46,6 +46,8 @@ completing D, E and F.  The course material is here, but see `our
 exercise page <https://plus.cs.aalto.fi/CS-E4004/2020/>`__ for
 information on credits.
 
+Attention! If your home university is not Aalto university, you must have flexible study 
+rights to get credits. Contact your home university for more information.
```

Comment:
```suggestion
Attention! If your home university is not Aalto University, but you are in a university in Finland, you have flexible study 
rights to get credits directly from Aalto. Contact your home university for more information: search "JOO", "joustava opinto-oikeus".
```
What about this clarification?  I don't know how well known this is but if I was new, I wouldn't know about this yet....